src/fprime/util/build_helper.py

```diff
@@ -255,19 +129,16 @@ def utility_entry(args):
                 skip_validation=does_not_need_cache_directory,
             )
         validate_tools_from_requirements(build)
-        status = runners[parsed.command](
-            build, parsed, cmake_args, make_args, getattr(parsed, "pass_through", [])
-        )
     except GenerateException as genex:
         print(
             f"[ERROR] {genex}. Partial build cache remains. Run purge to clean-up.",
             file=sys.stderr,
         )
-        return genex.exit_code
+        sys.exit(1)
```

Comment:
Should these exception traps be moved to cli?  I typically don't like to se sys.exit() so deep in the code.